CONTRIBUTORS.md

```diff
@@ -32,52 +32,52 @@ If you would like to contribute to the HELICS project see [CONTRIBUTING](CONTRIB
 ## Used Libraries or Code
 
 ### [BOOST](https://www.boost.org)
-  Boost is used throughout the code.  The unit, integration, and system test suite is written using boost test.  The IPC core uses the boost interprocess library, the program options library is used for parsing command line options and the filesystem library is used frequently when file manipulation is needed.  Some algorithms and other libraries are also used throughout the code. Boost is licensed under the boost license
+  Boost is used throughout the code. The unit, integration, and system test suite is use the Boost.Test library. The IPC core uses the Boost.Interprocess library. Some of the header-only Boost algorithms and other libraries are also used throughout the code. Boost is licensed under the Boost Software License.
 
 ### [Asio](https://think-async.com/Asio)
-  Asio is used for TCP and UDP communication, as well as resolving IP addresses. The asio library is included as a submodule Asio is licensed under the [Boost Software License](https://github.com/chriskohlhoff/asio/blob/master/asio/LICENSE_1_0.txt).
+  Asio is used for TCP and UDP communication, as well as resolving IP addresses. The Asio library is included as a submodule. Asio is licensed under the [Boost Software License](https://github.com/chriskohlhoff/asio/blob/master/asio/LICENSE_1_0.txt).
 
 ### [cppzmq](https://github.com/zeromq/cppzmq)
-  The header only bindings for the ZeroMQ library are used to interact with the ZeroMQ library the header files are modified to include some string operations and are included in HELICS source.  Cppzmp is licensed under the [MIT](https://github.com/zeromq/cppzmq/blob/master/LICENSE) license.
+  The header only bindings for the ZeroMQ library are used to interact with the ZeroMQ library. The header files are modified to include some string operations and are included in the HELICS source. cppzmq is licensed under the [MIT](https://github.com/zeromq/cppzmq/blob/master/LICENSE) license.
 
-### [jsoncpp](https://github.com/open-source-parsers/jsoncpp)
-  JsonCPP is used for parsing json files, it was chosen for easy inclusion in the project and its support for comments. Jsoncpp is licensed under public domain or MIT in case public domain is not recognized [LICENSE](https://github.com/open-source-parsers/jsoncpp/blob/master/LICENSE)
+### [JsonCpp](https://github.com/open-source-parsers/jsoncpp)
+  JsonCp is used for parsing json files, it was chosen for easy inclusion in the project and its support for comments. JsonCpp is licensed under public domain or MIT in case public domain is not recognized [LICENSE](https://github.com/open-source-parsers/jsoncpp/blob/master/LICENSE).
```

Comment:
```suggestion
  JsonCpp is used for parsing json files, It is included as a submodule from a slightly modified fork to add support for object libraries and to clean up some warning issues. JsonCpp is licensed under public domain or MIT in case public domain is not recognized [LICENSE](https://github.com/open-source-parsers/jsoncpp/blob/master/LICENSE).
```